README.md

```diff
@@ -1746,8 +1736,6 @@ Bookmarks
 
 [Georgia Tech - High Performance Computer Architecture - Udacity](https://classroom.udacity.com/courses/ud007)
 
-[MIT - Comp Org, Arch, and Digital Design](https://6004.mit.edu/web/spring19/course_information)
```

Comment:
I think this would be a suitable replacement: https://ocw.mit.edu/courses/6-004-computation-structures-spring-2017/